### PR TITLE
Add field support for dataset creation

### PR DESCRIPTION
**Pull request by roaraya8**

### CODE REVIEW

File: marquez_client/client.py [hunk 1]
```diff
@@ -306,6 +310,25 @@ def list_datasets(self, namespace_name=None, limit=None, offset=None):
             }
         )
 
+    @staticmethod
+    def make_field(name, data_type, description=None):
```
Comment (wslulciuc) [inline marquez_client/client.py]: What's the usage of `make_field`?

Comment (wslulciuc) [reply marquez_client/client.py]: _Minor_: We may want to add this helper method in a `utils.py`, but not a blocker to merge

**Review by wslulciuc — APPROVED** [review]
Left a minor comment, otherwise 👍

File: marquez_client/client.py (hunk 1 above)
Comment (roaraya8) [reply marquez_client/client.py]: It's mainly used as a helper method when collecting metadata and check for supported field types.
Sounds good! Will move it to a `utils.py` package

**Pull request merged**